packages/design-system/src/components/Table/TableCell.test.jsx

```diff
@@ -7,30 +7,24 @@ import TableRow from './TableRow';
 import { mount } from 'enzyme';
 
 const defaultTableHeadChildren = (
```

Comment:
Same here, unit tests should be in isolation. If you want to ensure that `TableCell` is rendering correctly based off its parent element, you should be testing `_isTableHeadChild` in `TableCell.test.jsx`, and also testing that that prop is being set and passed onto the children correctly in `TableBody.test.jsx` and `TableHead.test.jsx`